recipes/azure-identity-cpp/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "azure-identity-cpp" %}
+{% set version = "1.5.1" %}
+{% set sha256 = "651a660592540c267420b7e1c8dc98428a17396e45f7f6c92a864ab9a4fa028a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Azure/azure-sdk-for-cpp/archive/refs/tags/azure-identity_{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler("cxx") }}
+    - cmake
+    - make  # [not win]
+    - ninja  # [win]
+  host:
+    - azure-core-cpp >=1.9.0
```

Comment:
I think we'll have to add this to the pinning, and in this case the `>=` will be detrimental to having this handled by the bot.